packages/core/src/utils/transformers/media/getOptimizedBackgroundImageAsset.ts

```diff
@@ -1,50 +1,37 @@
 import { SUPPORTED_IMAGE_FORMATS } from '@/constants';
 import { OptimizedBackgroundImageAsset } from '@/types';
+import { getOptimizedImageUrl } from './getOptimizedImageUrl';
 import { AssetFile } from 'contentful';
 
 interface AssetFileWithRequiredImage extends AssetFile {
   details: Required<AssetFile['details']>;
 }
```

Comment:
BTW, I noticed there's an `ImageFile` type in the `contentful` package that includes the types for `details` and `details.image`

Reply:
lies

Reply:
It turns out `ImageFile` in in a private types packages that's used in the web app.  Nevermind this idea